src/__test__/table-theme-colors.spec.js

```diff
@@ -0,0 +1,296 @@
+import tableThemeColors from '../table-theme-colors';
+
+describe('table-theme-colors', () => {
+  const spyQuerySelector = jest.fn();
+  const spyGetComputedStyle = jest.fn();
+  const spyGetStyle = jest.fn();
+  const setObjectAndTableBackgroundColorInTheme = (objectBackgroundColor, tableBackgroundColor) =>
+    jest.fn((base, path, attribute) => {
```

Comment:
same thing here or does it have a purpose?

Reply:
same thing. No. 
I missed it. But now it is covered.